roles/letsencrypt/tasks/install-Debian.yml

```diff
@@ -0,0 +1,29 @@
+---
+
+- name: "Required packages"
+  yum:
+    name:
+      - snapd
+      - python-simplejson
+    state: present
+
+- name: Ensure versions of certbot from apt are not present
+  apt:
+    name:
+      - certbot
+    state: absent
+
+- name: snap install core
+  snap:
+    name: core
+
+- name: snap install certbot
+  snap:
+    name: certbot
+    classic: yes
```

Comment:
Check if this is really required in Debian 10. It should be sufficient to apt certbot

Reply:
Following the official documentation
[https://certbot.eff.org/lets-encrypt/debianbuster-nginx](https://certbot.eff.org/lets-encrypt/debianbuster-nginx)
it sais that we should install from snap.
Moreover, on step (4) it sais that we should purge any versions of certbot OS repository, if already installed !
I really don't know why it asks for that. Maybe the debian packages are very old?

Reply:
I added some more tasks and a flag. Now you can pick either snap or the distribution repo as an installation source.
The flag is "install_from" with values { "snap" | "os_repo" }